onelogin_aws_cli/argparse.py

```diff
@@ -17,7 +17,7 @@ def __init__(self):
         self.add_argument(
             '-C', '--config-name',
             action=EnvDefault, required=False,
-            dest='config_name',
+            dest='config_name', default='defaults',
```

Comment:
The stumbling block I ran into is that when you do it this way, and you try to retrieve the section, per configparser, the default section is not acknowledged by has_section so we'll try to add it.  However, adding a section with the name of the default section throws an exception.

Reply:
We can change it so it errors or returns `None` or something if there's no section. It doesn't need to be that smart.

If no section exists, we can make it explicit that a section would need to be added. Would that fix it?

Reply:
That would but then it just moves a check somewhere else (see line 88 of my PR)

Reply:
Can you link to it? I can not see a line 88 in the diffs.

Reply:
Maybe moving the check is a good thing.

I propose we should only create a section when we run the `--configure` option. Any other time we should (gracefully) fail and say we don't have the section.

That is, only create a section when we explicitly want to create a new one. Any other time, say we don't have one.

Reply:
I think I've got this working now.